src/main/scala/concurrency/FrequencyHistogram.scala

```diff
@@ -0,0 +1,23 @@
+package concurrency
+
+import java.io.{BufferedWriter, File, FileWriter}
+import scala.collection.concurrent.TrieMap
+import scala.collection.parallel.CollectionConverters.ImmutableIterableIsParallelizable
+import scala.io.Source
+
+object FrequencyHistogram extends App {
+  val src = Source.fromResource("t8.shakespeare.txt")
+  val wordCount = TrieMap[String, Int]()
+
+  src.getLines.toList.par
+    .flatMap(_.split("[\\s.,:;!?()'\"\\[\\]]+").map(_.toLowerCase()))
+    .foreach(w => wordCount.updateWith(w) {
+      case Some(c) => Some(c + 1)
+      case None => Some(1)
+    })
+
+  val histogram = for (i <- wordCount) yield f"${i._1}%-20s${"*" * i._2}"
```

Comment:
It's better to use more readable naming instead of i._1, i._2. E.g.  :
`for ((word, count) <- wordCount) yield f"${word%-20s${"*" * count}"`

Reply:
changed the names